src/lib/Repository/Values/Content/UiVersionInfo.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository\Values\Content;
+
+use eZ\Publish\Core\Repository\Values\Content\VersionInfo;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo as APIVersionInfo;
+
+/**
+ * Extends original value object in order to provide additional fields.
+ * Takes a standard VersionInfo instance and retrieves properties from it in addition to the provided properties.
```

Comment:
Added 👍 I'll go through and create a separate PR that will add it to the other VO